chain/txValidator.go

```diff
@@ -9,11 +9,13 @@ import (
 
 	"github.com/nknorg/nkn/block"
 	. "github.com/nknorg/nkn/common"
+	"github.com/nknorg/nkn/crypto"
 	. "github.com/nknorg/nkn/pb"
 	. "github.com/nknorg/nkn/transaction"
 	"github.com/nknorg/nkn/util/address"
 	"github.com/nknorg/nkn/util/config"
 	"github.com/nknorg/nkn/vm/signature"
+	"github.com/ontology/common"
```

Comment:
I think this is probably a typo

Reply:
I believe this is generated automatically by goimports. Every time I use log.xxx it will import nnet/log for me 😂